src/configurations.ts

```diff
@@ -122,6 +122,14 @@ const configurations = {
         configSalt: readStringValue('CONFIG_SALT', ''),
         autoDeleteTemp: readBooleanValue('AUTO_DELETE_TEMP_FILES', true),
     },
+    monitoring: {
+        memory: {
```

Comment:
Is there a way to disable memory monitoring?

Reply:
Set all levels above 100